pkg/virt-launcher/virtwrap/live-migration-target.go

```diff
@@ -20,32 +20,107 @@
 package virtwrap
 
 import (
+	"encoding/xml"
 	"fmt"
 	"net"
 	"path/filepath"
 	"strconv"
+	"time"
 
 	v1 "kubevirt.io/api/core/v1"
 	"kubevirt.io/client-go/log"
 
 	diskutils "kubevirt.io/kubevirt/pkg/ephemeral-disk-utils"
 	cmdv1 "kubevirt.io/kubevirt/pkg/handler-launcher-com/cmd/v1"
 	"kubevirt.io/kubevirt/pkg/hooks"
+	netvmispec "kubevirt.io/kubevirt/pkg/network/vmispec"
 	"kubevirt.io/kubevirt/pkg/util"
 	"kubevirt.io/kubevirt/pkg/util/net/ip"
 	migrationproxy "kubevirt.io/kubevirt/pkg/virt-handler/migration-proxy"
 	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api"
+	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/cli"
 	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/converter"
 )
 
-func (l *LibvirtDomainManager) finalizeMigrationTarget(vmi *v1.VirtualMachineInstance) error {
+func (l *LibvirtDomainManager) finalizeMigrationTarget(vmi *v1.VirtualMachineInstance, options *cmdv1.VirtualMachineOptions) error {
+	interfaceMigrationOptions := options.InterfaceMigration
+	if err := l.reconnectGuestNics(vmi, interfaceMigrationOptions); err != nil {
+		return err
+	}
+
 	if err := l.setGuestTime(vmi); err != nil {
 		return err
 	}
 
 	return nil
 }
 
+// Set interfaces link down and up to renew DHCP leases
+func (l *LibvirtDomainManager) reconnectGuestNics(vmi *v1.VirtualMachineInstance, interfaceMigrationOptions map[string]*cmdv1.InterfaceBindingMigration) error {
+	domName := api.VMINamespaceKeyFunc(vmi)
+	dom, err := l.virConn.LookupDomainByName(domName)
```

Comment:
Dont we need to set the `domainModifyLock`?
https://github.com/kubevirt/kubevirt/blob/e4da573b08ca9238d1e6d23675ed5602f0919f57/pkg/virt-launcher/virtwrap/manager.go#L148-L149